grid/src/Cabana_Grid_BovWriter.hpp

```diff
@@ -172,16 +172,17 @@ reorderView( TargetView& target, const SourceView& source,
   This version writes a single output and does not use bricklets. We will do
   this in the future to improve parallel visualization.
 
+  \param prefix The filename prefix
   \param time_step_index The index of the time step we are writing.
   \param time The current time
   \param array The array to write
   \param gather_array Gather the array before writing to make parallel
   consistent.
 */
 template <class ExecutionSpace, class Array_t>
-void writeTimeStep( ExecutionSpace, const int time_step_index,
-                    const double time, const Array_t& array,
-                    const bool gather_array = true )
+void writeTimeStep( ExecutionSpace, const std::string& prefix,
```

Comment:
We'll want a backwards compatibility wrapper that passes an empty string to this function (that we deprecate)

Reply:
I tried to create this (instead of passing an empty string, the deprecated version replicates the previous behavior of creating a prefix based on the array label).  However, there's an issue stemming from there already being another version of this function that infers the ExecutionSpace:
When one (like in the test cases) calls `writeTimeStep( "grid_cell_field_2d", 302, 3.43, *cell_field )`, it should be going to the new, non-deprecated method, but the compiler instead matches it to the templated `writeTimeStep( ExecutionSpace, const int...)`, templating ExecutionSpace to std::string.
I don't know how to fix this; I tried some stuff that would say "template ExecutionSpace to any class but std::string" but couldn't get it to work.

Reply:
We'll just need to add an `enable_if` to restrict that first argument to a valid execution space - we can chat about it Monday